### ISSUE TITLE
Cannot drag selected text and drop it somewhere else in editor, e.g. reorder sentences.

### ISSUE THREAD

**Issue by petrofcikmatus** [typora/typora-issues#827]
Typora version: 0.9.31 (beta)
OS version: Ubuntu Gnome 16.04.3
Description: I cannot drag selected text to another position. I wanted to reorder some sentences, but no luck. I saw "dragging shadow" but after dropping text, nothing happened.
How to reproduce: Select text and drag it somewhere else. You will see "draggin shadow" but after you drop text, nothing happens.

**Comment by vr8hub** [comment]
This appears to be an overall Typora problem, not just a Typora on Linux problem. It doesn't work on MacOS .9.9.13.6, either.

**Comment by stanstrup** [comment]
Neither on windows.

**Comment by leowill01** [comment]
This is essential for me. I'm not sure why Typora doesn't support it because it seems like a basic text editing feature. I would use this many times a day.

**Comment by denisvanw** [comment]
I also would love to be able to use a standard drag and drop ! 
I understand it might not be a coder's request...
But I think Typora could also be a great tool for writing.

**Comment by Kwen-Chen** [comment]
So in 2024, Typora has started a paid system, why don't you add this feature

**Comment by sleepdontsmile** [comment]
> So in 2024, Typora has started a paid system, why don't you add this feature

Exactly. The only thing that stops me from paying for Typora - the lack of this single feature disqualifies Typora from being an everyday editor for me.
